src/publish.ts

```diff
@@ -43,7 +43,11 @@ export function publish(data: TDocletDb, opts: ITemplateConfig)
 
         const pkgArray: any = helper.find(data, { kind: 'package' }) || [];
         const pkg = pkgArray[0] as IPackageDoclet;
-        const out = path.join(opts.destination, opts.outFile || (pkg && pkg.name ? `${pkg.name}.d.ts` : 'types.d.ts'));
+        let definitionName = 'types';
+        if (pkg && pkg.name) {
+          [definitionName] = pkg.name.split('/').reverse();
```

Comment:
I think it would be simpler to just be `definitionName = pkg.name.split('/').pop();`. No destructuring necessary.

Reply:
Good point; will change it

Reply:
Btw, I only added in the trailing `||` condition for the TS linter, but I'm sure that there won't ever be a condition where it'll be evaluated. I can put in a linting ignore comment if you prefer that though.